src/Controller/CreateModuleController.php

```diff
@@ -46,7 +48,13 @@ public function index(Request $request): Response
                 $newImage = $imageManager->createImage($request->files->get('create_module')['image'], $user, $this->getParameter('uploads_directory'), 'module');
                 $this->flushNewImage($newImage);
                 $module->setImage($newImage->getSrc());
-                $this->flushNewModule($module);
+
+                $module = $this->flushNewModule($module);
```

Comment:
Is there a reason why you flush the Module first, and then the cat?
You can do all of that in one go?

Reply:
fixed! thanks!